packages/main/src/RangeSlider.js

```diff
@@ -678,10 +721,6 @@ class RangeSlider extends SliderBase {
 		return this.getDomRef().querySelector(".ui5-slider-progress");
 	}
 
-	get tabIndexProgress() {
-		return this.tabIndex;
-	}
-
```

Comment:
After removing of this code, the _tabindex private property declared in SliderBase.js is no longer used and not needed, remove it also.

Reply:
Also in SliderBase.js remove the tabIndex getter from the bottom which is also no longer used anymore